tool-plugins/intellij/build.gradle

```diff
@@ -35,6 +35,8 @@ dependencies {
     compile group: 'org.eclipse.lsp4j', name: 'org.eclipse.lsp4j', version: '0.6.0'
     compile group: 'com.vladsch.flexmark', name: 'flexmark', version: '0.34.58'
     compile 'com.github.ballerina-platform:lsp4intellij:master-SNAPSHOT'
```

Comment:
This is not related to this PR, but the format of this dependency declaration seems different from others. Shall we update this so that all dependency declarations follow the same convention?

Reply:
+1 :) Updated the dependency format.